### DISCUSSION TITLE
What is the best way to recursively download everything over multiple runs?

### DISCUSSION THREAD

**Discussion by xave** [Serene-Arc/bulk-downloader-for-reddit#882] [Q&A]
Seeing that there is a limit of 1000, my idea is to run the command multiple times while using the `no-dupes`. 

The idea is this (simplified)

```
First run downloads:
Post 1
Post 2
...
Post 1000

Second run skips 1-1000 and downloads:
Post 1001
Post 1002
...
Post 2000

Etc.
Etc.
```

In essence, what is the strategy for the most complete download of data (saved, upvotes, comments, etc.  + context)?

**Comment by Fakeaccount12312** [comment]
This won't work since the 1000 post limit includes the files skipped with no-dupes, as these still have to be requested in order to verify these are duplicates. If you want to download the posts related to your account, the best idea ist to request your personal data from Reddit (they are required to provide this possibility to EU and California citizens, and everyone else can do it too). After some time, you will get a zip file with various csv files with all your data. Take the one with the links to the posts you want (saved etc..) and follow the strategy descibed [here](https://github.com/aliparlakci/bulk-downloader-for-reddit/discussions/877#discussioncomment-6144696).

I don't have any personal experience with this as neither the amount of my saved posts nor created posts surpasses 1000, and I made so many comments it would take ages to download them, but I don't really care too much about them anyways.

**Comment by ericcherny** [comment]
Is the question how to access post 1001 (and onward)? What if we tried to use common dictionary words and scrape using the search feature with `--no-dupes`?

**Reply by Serene-Arc** [reply]
You can't get past 1000 posts using any method bar the data request. Reddit simply doesn't allow it.

**Reply by SplitMilky** [reply]
Can you get their manually via the browser?
